src/main/java/org/jqassistant/contrib/plugin/php/scanner/parser/PHPFunctionParser.java

```diff
@@ -121,6 +153,11 @@ else if (tree.getClass().getSimpleName().equals("BlockStatementContext")){
         }
     }
     
+    /**
```

Comment:
This documentation does neither fit the method name nor the parameters. Please rename the function or change the documentation. It has to be consistent.